plugins/flashrom/fu-reset-cmos.c

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (C) 2021 Sean Rhodes <sean@starlabs.systems>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+#include "config.h"
+
+#include <libflashrom.h>
+#include <sys/io.h>
+#include <unistd.h>
```

Comment:
I don't think you need this header do you?

Reply:
Which bit? 

```
#include <sys/io.h>
#include <unistd.h>
```
These are needed for ioperm

Reply:
sys/io sure, but isn't unistd just for adding stuff like bool?  Which you don't need since you're using gboolean.
I guess double check on your side with a local compile test.  If it works drop it, if it doesn't leave it and we'll resolve this one.

Reply:
Fair point.

if I pull it, I get:
```
● fwupd.service - Firmware update daemon
     Loaded: loaded (/lib/systemd/system/fwupd.service; static)
     Active: failed (Result: signal) since Fri 2021-09-10 15:18:27 BST; 7s ago
       Docs: https://fwupd.org/
    Process: 3123 ExecStart=/usr/libexec/fwupd/fwupd (code=killed, signal=SEGV)
   Main PID: 3123 (code=killed, signal=SEGV)

Sep 10 15:18:27 sean-StarBook fwupd[3123]: 14:18:27:0022 FuDevice             no FuContext assigned for FuFlashromInternalDevice:
Sep 10 15:18:27 sean-StarBook fwupd[3123]: Unknown Device
Sep 10 15:18:27 sean-StarBook fwupd[3123]:   Protocol:             org.flashrom
Sep 10 15:18:27 sean-StarBook fwupd[3123]:   Flags:                internal|updatable|require-ac|needs-reboot
Sep 10 15:18:27 sean-StarBook fwupd[3123]: 14:18:27:0022 FuDevice             7 is not a valid triplet (guessed number)
Sep 10 15:18:27 sean-StarBook fwupd[3123]: 14:18:27:0026 GLib-GObject         g_object_new_valist: assertion 'G_TYPE_IS_OBJECT (object_type)' failed
Sep 10 15:18:27 sean-StarBook fwupd[3123]: 14:18:27:0026 FuDevice             fu_device_add_instance_id: assertion 'FU_IS_DEVICE(self)' failed
Sep 10 15:18:27 sean-StarBook systemd[1]: fwupd.service: Main process exited, code=killed, status=11/SEGV
Sep 10 15:18:27 sean-StarBook systemd[1]: fwupd.service: Failed with result 'signal'.
Sep 10 15:18:27 sean-StarBook systemd[1]: Failed to start Firmware update daemon.

```

Which doesn't really make sense, could be a red herring

Reply:
The failure looks like it could maybe happen if you are loading stuff from /usr/local and from /usr/lib, I don't think it's related to this.